tests/test_vlmeta.py

```diff
@@ -59,12 +59,12 @@ def test_schunk(contiguous, urlpath, nbytes, cparams, dparams, nchunks):
 
 
 def add(schunk):
-    schunk.vlmeta["vlmeta1"] = "val1"
-    schunk.vlmeta["vlmeta2"] = "val2"
-    schunk.vlmeta["vlmeta3"] = "val3"
-    assert schunk.vlmeta["vlmeta1"] == "val1"
-    assert schunk.vlmeta["vlmeta2"] == "val2"
-    assert schunk.vlmeta["vlmeta3"] == "val3"
+    schunk.vlmeta["vlmeta1"] = b"val1"
```

Comment:
I suppose this is how msgpack behaves when doing a roundtrip of a string, right?  In that case, better let it as it is now.

Reply:
msg pack does maintain the original type of the vlmeta  value when deserializing, so if it is a string it will return a string and if it is a byte string it will return a byte string.